src/ddocs/views/intro.rst

```diff
@@ -87,10 +87,18 @@ familiar with JavaScript but have used C or any other C-like language such as
 Java, PHP, or C#, this should look familiar. It is a simple function definition.
 
 You provide CouchDB with view functions as strings stored inside the ``views``
-field of a design document. You don’t run it yourself. Instead, when you
+field of a design document. To create this view you can use this command:
+
```

Comment:
It might be better to use a code block here. As in:

```
.. code-block:: text

       $ curl -X PUT http://admin:password@127.0.0.1:5984/db/_design/my_ddoc \
         -d '{"views":{"my_filter":{"map":
          "function(doc) { if(doc.date && doc.title) { emit(doc.date, doc.title); }}"}}}'
```

So it renders just like to other examples in the page

Reply:
done. I did not put in the `$`. Is there a default this?
My grep showed that's sometimes with $ sometimes without.